topics/digital-humanities/tutorials/introduction_to_dh/tutorial.md

```diff
@@ -0,0 +1,572 @@
+tags:
+- text mining
+contributions:
+  authorship:
+    - Sch-Da
+  funding:
+    - deKCD
+---
+
+<!-- This is a comment. -->
+
+Loosely building on {% cite Richardson2003 %},[^1] this tutorial compares two editions of the poem "The Sorrows of Yamba".
+The first couple of steps in this tutorial derive from [A short introduction to Galaxy]({% link topics/introduction/tutorials/galaxy-intro-short/tutorial.html %}).
+
+"The Sorrows of Yamba" was published in 1795 and was among the most popular antislavery poems at the time. However, the version published by Hannah More in the Cheap Repository Tracts series was not the only version of the poem that circulated. Also, Moore's authorship on the topic is not uncontested. {% cite Richardson2003 %} But while we leave this debate to the experts, the different versions of the poem offer a great opportunity to delve into how digital tools can help us compare texts more quickly. And this, we will do in the following tutorial.
+
+While Richardson compared the poems by hand, we use his example to introduce how Galaxy can help you with your text analysis.
+From logging in, uploading the texts, to using the first tools, this tutorial covers the Galaxy basics.
+We will clean the two poem versions and check the texts from a distance by comparing their number of lines and characters and visualising both in a word cloud. 
+Then, we take a closer look. For an easier comparison, we reformat both texts and compare them line by line and side by side. As the word cloud shows, "death" is a dominant theme in the first poem, so we extract all lines including "death" for further in-depth analysis. This helps us get a better idea of where those articles differ and is applicable to many other texts you might want to compare.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Hands-on Sections
+
+## Create an account on Galaxy
+To use Galaxy's full potential, you need to register and create an account. You can skip this step if you already have a Galaxy account.
+
+{% snippet faqs/galaxy/account_create.md %}
+
+Alternatively, you can log in using a single sign-on of your choice, for example, from [IAM4NFDI](https://iam.services.base4nfdi.de/faq_ENG/) on [Galaxy Europe](https://usegalaxy.eu/).
+
+ ![Screenshot of Galaxy Europe register window with the IAM4NFDI login button highlighted](../../images/iam4nfdi.png)
+
+## Log in to Galaxy
+<!-- to do adapt screenshot / upload / make suitable 4 Europe -->
+> <hands-on-title>Log in to Galaxy</hands-on-title>
+> 1. Open your favourite browser (Chrome, Safari, Edge or Firefox as your browser, not Internet Explorer!)
+> 2. Browse to your Galaxy instance, for example [Galaxy Europe](https://usegalaxy.eu/)
+> 3. Log in with your credentials
+>  
+> ![Screenshot of Galaxy Australia with the register or login button highlighted](../../images/galaxy-login.png)
```

Comment:
for all these images that come from the introduction topic, you should change it to provide the full path, like this:

```suggestion
>  ![upload data button shown in the galaxy interface]({% link topics/introduction/images/upload-data.png %})
```